### ISSUE TITLE
Prompt showing only once

### ISSUE THREAD

**Issue by Karjan1** [larryaasen/upgrader#164]
I try to force users to upgrade app each time there is a new version. It works but only once, when there is a new version. When the dialog is opened and the user closes app. It won't ask again for update. Is there an option to prompt dialog each time?

**Comment by larryaasen** [comment]
https://github.com/larryaasen/upgrader#customization
`durationUntilAlertAgain: duration until alerting user again, which defaults to 3 days`

**Comment by larryaasen** [comment]
@Karjan1 Did that work for you?

**Comment by Karjan1** [comment]
Yes, after upgrading to new version and using the param you provided. All works fine. Sorry for the delay
